composer.json

```
@@ -14,21 +14,21 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": "^8.0.2",
```

After all these years I still get confused with this syntax :). I thought it should be:

```suggestion
        "php": "^8.0|^8.1",
```

but composer doesn't complain in 8.1 tests, so I guess this is fine. Also, why 8.0.2 specifically?

---

It is what [Laravel 9 uses as requirement](https://github.com/laravel/framework/blob/9.x/composer.json#L18), and [according to Composer](https://getcomposer.org/doc/articles/versions.md#caret-version-range-), the recommended operator for maximum interoperability when writing library code.

This will allow this package to be installed in PHP `8.0`, `8.1`, `8.2` etc, without the need of updating our requirements.